qemu/tests/cfg/qemu_img.cfg

```diff
@@ -12,7 +12,7 @@
             remove_image_dd = yes
             create_image_cmd = "dd if=/dev/zero of=%s bs=1G count=1"
             # Test the conversion from 'dd_image_name' to specified format
-            supported_image_formats = qcow2 raw qed
+            supported_image_formats = qcow2 raw
             Host_RHEL.m5:
```

Comment:
As supported_image_formats is same, no need to separate it for RHEL-5 host. Drop the follow two lines.

Reply:
Yes, thanks.